### ISSUE TITLE
server is hardcoded in authz_server APIs

### ISSUE THREAD

**Issue by matteo-cristino** [ForkbombEu/DIDroom_microservices#43]
In all athz_server APIs, in the keys the `server` variable is hardocded as
```json
{
  "server": {
    "url": "https://authz-server1.zenswarm.forkbomb.eu:3100",
    "authentication_url": "https://did.dyne.org/dids/"
  }
}
```
at least the `url` should be read from the well-known or specified at time of starting the server. The first seems the best option, but it should be checked if the authz_server can contain a variable in which it is stored. For the moment only the endpoints are present in the well-known, not the base url.

**Comment by matteo-cristino** [comment]
looking at RFC 8414 in particular at [section 2](https://datatracker.ietf.org/doc/html/rfc8414#section-2) it seems that the `issuer` key in the authz_server well known should have as value the url of the `authz_server` not the one of the `credential issuer`

**Comment by andrea-dintino** [comment]
Fixed .well-known of the authz-server in https://github.com/ForkbombEu/DIDroom_microservices/commit/91522198a8129443760f6802765d7d005a0e13e8

**Comment by matteo-cristino** [comment]
APIs fixed in #47
